src/streamlink/plugins/bloomberg.py

```diff
@@ -32,10 +32,9 @@ class Bloomberg(Plugin):
 ''', re.VERBOSE)
     _live_player_re = re.compile(r'{APP_BUNDLE:"(?P<live_player_url>.+?/app.js)"')
     _js_to_json_re = partial(re.compile(r'(\w+):(["\']|\d?\.?\d+,|true|false|\[|{)').sub, r'"\1":\2')
-    _video_id_re = re.compile(r'data-bmmr-id=\\"(?P<video_id>.+?)\\"')
     _mp4_bitrate_re = re.compile(r'.*_(?P<bitrate>[0-9]+)\.mp4')
-    _preload_state_re = re.compile(r'window.__PRELOADED_STATE__\s*=\s*({.*});', re.DOTALL)
-    _live_stream_info_re = re.compile(r'12:.*t.exports=({.*})},{}],13:', re.DOTALL)
+    _preload_state_re = re.compile(r'<script>\s*window.__PRELOADED_STATE__\s*=\s*({.+});?\s*</script>')
+    _live_stream_info_re = re.compile(r'],212:\[function\(.+var n=({.+});r.default=n},{"[^"]+":\d+}],213:\[', re.DOTALL)
```

Comment:
> Extracting this data will always be problematic,

already outdated, it is **213** now

the streams seems to have static URLs, might need to use them instead of this.